test/torch_test.py

```diff
@@ -610,6 +610,12 @@ def test_var_gradient_keeps_id_during_send_(self):
         assert model.data.id == original_data_id
         assert model.grad.data.id == original_grad_id
 
+    def test_operation_with_variable_and_parameter(self):
+        x = sy.Parameter(sy.FloatTensor([1]))
+        y = sy.Variable(sy.FloatTensor([1]))
+        z = x * y
+        assert torch.equal(z, sy.Variable(sy.FloatTensor([1])))
```

Comment:
Was this previously returning a Parameter or erroring out?

Reply:
Was erroring out!